client/components/Chat.vue

```diff
@@ -1,16 +1,18 @@
 <template>
-	<div id="chat-container" class="window" :data-current-channel="channel.name">
-		<div
-			id="chat"
-			:class="{
-				'hide-motd': !$store.state.settings.motd,
-				'colored-nicks': $store.state.settings.coloredNicks,
-				'show-seconds': $store.state.settings.showSeconds,
-			}"
-		>
+	<div
+		id="chat-container"
+		:class="{
+			window: true,
+			'hide-motd': !$store.state.settings.motd,
+			'colored-nicks': $store.state.settings.coloredNicks,
+			'show-seconds': $store.state.settings.showSeconds,
+		}"
+		:data-current-channel="channel.name"
+	>
+		<div id="chat">
```

Comment:
Remove this div?

Reply:
I guess we have to keep it as getting rid of it proved tricky.